requirements.txt

```diff
@@ -3,5 +3,7 @@ protobuf>=3.0
 pyaml==15.8.2
 numpy==1.11.0
 scikit-image==0.12.3
-tensorflow==0.9.0
+tensorflow==0.12.0
```

Comment:
@deshraj I am able to install this version through pip, but travis build is failing at this.